fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeHandler.java

```diff
@@ -1338,6 +1339,107 @@ public void updateTableMeta(Database db, String tableName, Map<String, String> p
         }
     }
 
+    // return true means that the modification of FEMeta is successful,
+    // and as long as the modification of metadata is successful,
+    // the final consistency will be achieved through the report handler
+    public boolean updateBinlogConfigMeta(Database db, Long tableId, Map<String, String> properties,
+                                          TTabletMetaType metaType) {
+        List<Partition> partitions = Lists.newArrayList();
+        OlapTable olapTable;
+        BinlogConfig newBinlogConfig;
+        boolean hasChanged = false;
+        boolean isModifiedSuccess = true;
```

Comment:
In java, it is better to throw an exception to the upper layer, and write stack log then use it to explain why the modification of the binlog failed. Instead of using isModifiedSuccess to judge.

Reply:
The reason for not throwing an exception directly is that ，this method modifies the FEMeta firstly, and send task to BE, as long as the FEMeta is modified successfully, it will return success, even if the be tasks fail and an exception is thrown。The configuration information will be synchronized through the reporthandler in the end